common/src/View/ControlListBox.cpp

```diff
@@ -247,15 +247,24 @@ namespace TrenchBroom {
         void ControlListBox::doubleClicked(const size_t /* index */) {}
 
         void ControlListBox::listItemSelectionChanged() {
+            bool wasAnyRowSelected = false;
+            
             for (int row = 0; row < count(); ++row) {
                 auto* listItem = m_listWidget->item(row);
                 auto* renderer = this->renderer(row);
+                // FIXME: this uses QListWidgetItem::isSelected() but addItemRenderer() is doing
```

Comment:
I don't like to merge PRs that contain fixmes or todos. Should this not be fixed right away? Otherwise I'm afraid it will never get fixed since there isn't any issue for it.

Reply:
I filed https://github.com/kduske/TrenchBroom/issues/3104

It's important to me to flag errors directly in the code as well as filing issues, because it's likely someone will look at this code again and have already forgotten about the issue, and the FIXME will remind them.

I didn't want to introduce new bugs as well by attempting to fix this as part of this PR.

Reply:
That's fine then. Maybe reference the issue from the comment so that it's clear that they are related.